core/domain/exp_domain.py

```diff
@@ -192,23 +223,6 @@ def __init__(self, change_dict):
         elif self.cmd == exp_models.ExplorationModel.CMD_REVERT_COMMIT:
```

Comment:
Hi @seanlip, I had a query here. The CMD_REVERT_COMMIT command is only being checked in ExplorationModels but not in other versioned models like collection and story. Is there any specific reason for that?

Reply:
I think it's probably because there's no actual user-visible functionality at the moment to revert collections and stories. It's probably easy enough to modify those manually to get back to the desired structure anyway, since they are basically just playlist.

I think for now let's just not support reverting for collections and stories? I think there's a flag ALLOW_REVERT that can be set to False for that.

Reply:
Ok, sounds good. Thanks for the explanation.